registration/tasks.py

```diff
@@ -259,6 +484,42 @@ def jembi_post_json(registration_id):
 
 
 @task()
+def jembi_post_xml(registration_id):
+    """ Task to send clinic & chw registrations XML to Jembi"""
+
+    logger.info("Compiling Jembi XML data")
+    try:
+        registration = Registration.objects.get(pk=registration_id)
+        xml_body = build_jembi_xml(registration)
+        json_body = build_json_body(xml_body, registration)
+        data = build_multipart_data("yolo", build_multipart_parts(json_body,
+                                                                  xml_body))
+
+        api_url = "%s/registration/net.ihe/DocumentDossier" % (
+            settings.JEMBI_BASE_URL)
+        headers = {
+            'Accept-Encoding': 'gzip',
+            'Content-Type': 'multipart/form-data; boundary=yolo'
+        }
+        auth = (settings.JEMBI_USERNAME, settings.JEMBI_PASSWORD)
+
+        try:
+            result = requests.post(api_url, headers=headers, data=data,
+                                   auth=auth, verify=False)
+            return result.text
```

Comment:
We need to capture the response and if it's successful mark the reg in the model at sent so we can requeue any issues